zrml/orderbook-v1/src/lib.rs

```diff
@@ -55,7 +55,7 @@ mod pallet {
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
-        #[pallet::weight(0)]
+        #[pallet::weight(10_000_000)]
```

Comment:
`#[pallet::weight(T::WeightInfo::cancel_order( ... ))]` ?

Reply:
@c410-f3r This pull requests only benchmarks and inserts weights for the swaps pallets.
For all other pallets, I have included reasonable default weights, so they can't be abused until we have proper benchmark and weights for them.

Reply:
Thanks for the clarification